udata/templates/dataset/display.html

```diff
@@ -387,7 +389,9 @@ <h3 class="text-left">{{ _('Producer') }}</h3>
         <div class="row">
             <div class="col-sm-9 list-group resources-list smaller">
             {% for resource in dataset.community_resources %}
-                {% include theme('dataset/resource/list-item.html') %}
+                <div itemprop="hasPart">
```

Comment:
I think this break the markup defined on resource list item which is already using Dataset/distribution aka. DataDownload
https://github.com/etalab/udata/blob/master/udata/templates/dataset/resource/list-item.html